content/about/notes/1.1/index.md

```diff
@@ -56,6 +56,11 @@ to create global namespaces. Locality-aware routing is enabled by default in the
 [Securing Kubernetes Ingress with Cert-Manager](/docs/examples/advanced-gateways/ingress-certmgr/) example for more details on how
 to use Kubernetes Ingress resources with [gateways](/docs/concepts/traffic-management/#gateways).
 
+- **Behavioral changes**.
+    - Outbound traffic policy is set to `ALLOW_ANY` by default. As a result, traffic to unknown ports will be forwarded as-is. Traffic to known ports (e.g., port 80) will be matched with one of the services in the system and forwarded accordingly.
+    - Destination rule resolution order has been set. For a given sidecar, when routing to a particular service, destination rules for the target service in the same namespace as the sidecar will first be considered, followed by destination rules in the service’s namespace, and finally followed by destination rules in `configRootNamespace` if defined or any other namespace.
+    - We recommend storing gateway resources in the same namespace as the gateway workload. E.g., `istio-system` in case of `istio-ingressgateway`. When referring to gateway resources in virtual services, use the `namespace/name` format instead of the `name.namespace.svc.cluster.local` format.
```

Comment:
I'm confused by this, because if I look at our ingress doc, we don't recommend users to deploy the gateway resource in the istio-system ns.  https://preliminary.istio.io/docs/tasks/traffic-management/ingress/#configuring-ingress-using-an-istio-gateway

Reply:
we should I guess.. so that in future we can make the full switch